core/shared/src/main/scala/scalaz/zio/ExitResult.scala

```diff
@@ -6,7 +6,7 @@ package scalaz.zio
  * completed with a value, failed because of an uncaught `E`, or terminated
  * due to interruption or runtime error.
  */
-sealed trait ExitResult[+E, +A] { self =>
+sealed abstract class ExitResult[+E, +A] extends Serializable { self =>
```

Comment:
I'd also extend `Product` together with `Serializable` for reasons [discussed elsewhere](https://typelevel.org/blog/2018/05/09/product-with-serializable.html).